.pre-commit-config.yaml

```
@@ -7,7 +7,7 @@ default_language_version:
   python: python3
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: 'v0.5.7'
+    rev: 'v0.6.2'
```

Bump to 0.6.3 instead and use consistent notation for versions (no quotes as in the pre-commit documentation):
```suggestion
    rev: v0.6.3
```